packages/parser/test/specs/circular/circular.test.ts

```diff
@@ -13,67 +12,41 @@ import validatedAPI from './validated.js';
 
 describe('API with circular (recursive) $refs', () => {
   it('should parse successfully', async () => {
-    const parser = new OpenAPIParser();
-    const api = await parser.parse(path.rel('specs/circular/circular.yaml'));
+    const api = await parse(relativePath('specs/circular/circular.yaml'));
 
-    expect(api).to.equal(parser.schema);
     expect(api).to.deep.equal(parsedAPI.api);
-    expect(parser.$refs.paths()).to.deep.equal([path.abs('specs/circular/circular.yaml')]);
   });
 
-  it(
```

Comment:
Removed these because our parser no longer has a resolve method, it just uses the internal `json-schema-ref-parser` one when we invoke methods in that library.